test_config/test_config.go

```diff
@@ -38,7 +38,6 @@ var ABIFilePath string
 
 func init() {
 	setTestConfig()
-	setInfuraConfig()
```

Comment:
Can `InufraClient` be removed in this file, since it looks like we're using `TestClient` now?

Reply:
Yes - nice catch!